Bottom: 62af8ac299f58105e2e1998f1fac3a8575ef3fe2
Top:    2377b6f81c8b0fcdc3e69870fb835e89121b808e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 15:55:50 +0100

Refresh of autoroute-c-simplify-some-long

---

diff --git a/src/autoroute.c b/src/autoroute.c
index 079c70b..554fda7 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -4845,11 +4845,8 @@ RouteAll (routedata_t * rd)
 			ras.conflict_subnets++;
 		      else
 			{
-			  double percent;
 			  ras.routed_subnets++;
 			  ras.total_nets_routed++;
-			  percent = calculate_progress (this_heap_item, this_heap_size, &ras);
-			  gui->progress (percent * 100., 100,  _("Autorouting tracks"));
 			}
 		    }
 		  else
